src/ObjectPool/src/ObjectPoolProvider.cs

```diff
@@ -3,13 +3,24 @@
 
 namespace Microsoft.Extensions.ObjectPool
 {
+    /// <summary>
+    /// A provider of <see cref="ObjectPool{T}"/>'s.
```

Comment:
Apostraphe is wrongzo, this is a pluralisation.

Reply:
In these instance's I tend to phrase it as `A provider of <Foo> instances.` thus avoiding the trap of incorrect apostrophization's.